app/views/merchant/dashboard/items.html.erb

```diff
@@ -0,0 +1,3 @@
+<% @items.each do |item| %>
+  <%= item.name %>
+<% end  %>
```

Comment:
I like that we added this just for a more robust test. Diligent!

Reply:
I wonder if more is coming for this page